src/entities/exportAccount.ts

```
@@ -0,0 +1,194 @@
+  }
+
+  async loadPosts() {
+    return db.query.posts.findMany({
+      where: eq(schema.posts.accountId, this.actorId),
+      with: getPostRelations(this.actorId), // Fetch related data using getPostRelations
+    });
+  }
+
+  async loadFollows(type: "following" | "followers") {
+    const column = type === "following" ? "followingId" : "followerId";
+    const column = type === "following" ? "followingId" : "followerId";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'column'. Consider to delete it or rename it.

'column' is defined here:


(lint/suspicious/noRedeclare)